aiohttp/web.py

```diff
@@ -221,8 +222,18 @@ def middlewares(self):
         return self._middlewares
 
     def make_handler(self, **kwargs):
-        return self._handler_factory(
-            self, self.router, loop=self.loop, **kwargs)
+        debug = kwargs.pop('debug', _sentinel)
```

Comment:
Raise `DeprecationWarning` if `debug` is not `_sentinel` like [here](https://github.com/KeepSafe/aiohttp/blob/1a64fa70b819998f165f3e8e1f0395048bb879b6/aiohttp/client.py#L685)

Also explicit test should check if the warning was raised like [here](https://github.com/KeepSafe/aiohttp/blob/471df5c4c9bf54a7843ae97c0d48b8182eda2609/tests/test_web_application.py#L66)